heartbeat/tomcat

```diff
@@ -56,7 +56,7 @@ else
   SU=su
 fi
 
-if ! ocf_is_true $OCF_RESKEY_force_script && which systemctl > /dev/null 2>&1; then
+if ! ocf_is_true $OCF_RESKEY_force_script && pidof systemd > /dev/null 2>&1; then
```

Comment:
The point is arbitrary user on the system can do:

```
echo 'int main(void){for(;;);}' |  gcc -osystemd -xc -; ./systemd
```

which the apparently an undesired interference between the unprivileged
user and the privileged process.